packages/block-editor/src/components/text-transform-control/stories/index.story.js

```diff
@@ -1,33 +1,74 @@
+/**
+ * Internal dependencies
+ */
+import TextTransformControl from '../';
+
 /**
  * WordPress dependencies
  */
 import { useState } from '@wordpress/element';
 
 /**
- * Internal dependencies
+ * Meta configuration for Storybook
  */
-import TextTransformControl from '../';
-
-export default {
+const meta = {
 	title: 'BlockEditor/TextTransformControl',
 	component: TextTransformControl,
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component:
+					'Control to facilitate text transformation selections.',
+			},
+		},
+	},
 	argTypes: {
-		onChange: { action: 'onChange' },
+		onChange: {
+			action: 'onChange',
+			control: {
+				type: null,
+			},
+			description: 'Callback function when text transform changes',
+			table: {
+				type: {
+					summary: 'function',
+				},
+			},
+		},
+		className: {
+			control: { type: 'text' },
+			description: 'Additional CSS class name to apply',
```

Comment:
```suggestion
			description: 'Class name to add to the control.',
```

Suggested change
			description: 'Callback function when text transform changes',
			description: 'Handles change in text transform selection.',
Use JSDoc comment